ui/app/templates/components/transit-key-action/decrypt.hbs

```diff
@@ -12,7 +12,7 @@
       <div id="ciphertext-control" class="control">
         <JsonEditor
```

Comment:
@chelshaw @hellobontempo something that didn't make sense to me at first, but is now, is why even if I don't remove `cipherText` from the props I carry over why the tabs were still not showing ciphertext. For the tabs that show ciphertext, we never pass in a `@value` to the json editor. So technically this means even if I don't remove cipherText from the props I don't show it. 

I am still going to try and remove ciphertext from `this.props` when navigating after rewrap, but it's kind of a safety we may previously built in.

Reply:
🤔 I wonder if the form would submit the previous ciphertext even though it's not shown if the value isn't updated (someone doesn't type in the json editor)... if that's the case (and I sort of assume it is) we should probably pass ciphertext to JsonEditor so it displays what the UI will use for the API call